docs/README.md

```diff
@@ -109,68 +109,71 @@ python3 runbenchmark.py --help
 ```
 
 ```text
-usage: runbenchmark.py [-h] [-m {local,aws,docker,singularity}]
-                       [-t [task_id [task_id ...]]]
-                       [-f [fold_num [fold_num ...]]] [-i input_dir]
-                       [-o output_dir] [-u user_dir] [-p parallel_jobs]
-                       [-s {auto,skip,force,only}] [-k [true|false]]
+usage: runbenchmark.py [-h] [-m {local,aws,docker,singularity}] [-t [task_id ...]] [-f [fold_num ...]] [-i input_dir] [-o output_dir] [-u user_dir] [-p parallel_jobs] [-s {auto,skip,force,only}]
```

Comment:
Oversight, sorry. Changed it, though the output renders slightly differently for me anyway (not sure if it is due to newer version of python (argparse) or because of MacOS.